java/code/src/com/redhat/rhn/taskomatic/task/sshservice/SSHServiceDriver.java

```diff
@@ -45,19 +45,19 @@
 import java.util.Set;
 
 /**
- * Call rhn_check on relevant systems via SSH using remote port forwarding.
+ * Provide services for salt ssh clients
  */
-public class SSHPushDriver implements QueueDriver<SystemSummary> {
+public class SSHServiceDriver implements QueueDriver<SystemSummary> {
 
     // Synchronized set of systems we are currently talking to
     private static final Set<SystemSummary> CURRENT_SYSTEMS = Collections.synchronizedSet(new HashSet<>());
 
     // String constants
     private static final String WORKER_THREADS_KEY = "taskomatic.ssh_push_workers";
     private static final String PORT_HTTPS_KEY = "ssh_push_port_https";
```

Comment:
We could create new keys for these, and use the old one as fallback, so the naming is consistent moving forward.